src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/DelegatingHandlerIntegration.cs

```diff
@@ -12,11 +12,11 @@
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNet
 {
     /// <summary>
-    /// System.Web.Http.Dispatcher.HttpRoutingDispatcher calltarget instrumentation
+    /// System.Net.Http.DelegatingHandler calltarget instrumentation
     /// </summary>
     [InstrumentMethod(
-        AssemblyName = "System.Web.Http",
-        TypeName = "System.Web.Http.Dispatcher.HttpRoutingDispatcher",
+        AssemblyName = "System.Net.Http",
+        TypeName = "System.Net.Http.DelegatingHandler",
```

Comment:
This seems a bit far reaching to me 🤔 `DelegatingHandler` makes up the pipeline of handlers in `HttpClient`, so we'd be instrumenting all of those too. I haven't looked to see if there's other options, but instrumenting this one concerns me somewhat 😟

Reply:
I understand your concern. Maybe we can add a new message handler to the beginning of the pipeline using automatic instrumentation. I'm going to experiment with that approach and see if it works

Reply:
I updated the instrumentation in 7c01174399bcaa4e67728a2a379b54340e8e7ac0 to instrument the exception handling pipeline instead, so it should be a much more constrained solution 👍🏼